examples/workflow/qe_scf_calculation.py

```diff
@@ -0,0 +1,187 @@
+ORGANIZATION_ID = "ORGANIZATION_ID"  # @param {type:"string"}
+
+import os
+
+if "COLAB_JUPYTER_IP" in os.environ:
+    os.environ.update(
+        dict(
+            ACCOUNT_ID=ACCOUNT_ID,
+            AUTH_TOKEN=AUTH_TOKEN,
+            MATERIALS_PROJECT_API_KEY=MATERIALS_PROJECT_API_KEY,
+            ORGANIZATION_ID=ORGANIZATION_ID,
+        )
+    )
+
+    get_ipython().system('GIT_BRANCH="dev"; export GIT_BRANCH; curl -s "https://raw.githubusercontent.com/Exabyte-io/api-examples/${GIT_BRANCH}/scripts/env.sh" | bash')
+
+
+# In[1]:
+
+
+from utils.settings import ENDPOINT_ARGS, ACCOUNT_ID
+from utils.generic import display_JSON
+
+from exabyte_api_client.endpoints.workflows import WorkflowEndpoints
+
+
+# In[2]:
+
+
+# Initialize a helper class to interact with WorkflowEndpoints
+endpoint = WorkflowEndpoints(*ENDPOINT_ARGS)
+
+
+# In[3]:
+
+
+# payload for workflow creation
+BODY = {
+    "name": "Silicon-SCF-bash-REST",
+    "subworkflows": [
+        {
+            "name": "QE-SCF",
+            "application": {
+                "name": "shell",
+                "version": "4.2.46",
+                "build": "Default",
+                "isDefault": "true",
+                "summary": "Shell Script",
+                "shortName": "sh",
+            },
+            "properties": ["total_energy", "fermi_energy"],
+            "units": [
+                {
+                    "type": "execution",
+                    "application": {
+                        "name": "shell",
+                        "version": "4.2.46",
+                        "build": "Default",
+                        "isDefault": "true",
+                        "summary": "Shell Script",
+                        "shortName": "sh",
+                    },
+                    "head": "true",
+                    "input": [
+                        {
+                            "content": "#!/bin/bash\n# ---------------------- CLUSTER PARAMETERS ---------------------- #\n#PBS -N Silicon-SCF\n#PBS -j oe\n#PBS -l nodes=1\n#PBS -l ppn=4\n#PBS -q OR\n#PBS -l walltime=00:00:30:00\n#PBS -A seminar-espresso-tutorials\n# ------------------------- INPUT FILES -------------------------- #\n# switch to the job working directory\ncd $PBS_O_WORKDIR\n\nBASE_URL='https://github.com/pranabdas/espresso/raw/next/src'\nIN_FILE='pw.scf.silicon.in'\nPP_FILE='Si.pz-vbc.UPF'\n\n# get QE input file\nwget ${BASE_URL}/silicon/${IN_FILE} -O ${IN_FILE}\n\n# get pseudopotential file\nwget ${BASE_URL}/pseudos/${PP_FILE} -O ${PP_FILE}\n\n# delete pseudo_dir from input file and provide via env\nsed -i '/^\\s*pseudo_dir/d' ${IN_FILE}\nexport ESPRESSO_PSEUDO='./'\n\n# --------------------------- RUN JOB ---------------------------- #\n# load required module\nmodule add espresso/63-i-174-impi-044\n\n# run the calculation\nmpirun -np $PBS_NP pw.x -in ${IN_FILE} | tee ${IN_FILE%*.in}.out\n\n# ----------------------------- END ------------------------------ #\n",
```

Comment:
The wall time is 30 minutes here but in the `compute` block it's set to `"timeLimit": "01:00:00"`. Probably should synchronize it.

Reply:
Also, this block has a content of a script file. We may want to think about the generation of the entry from a file.

Reply:
I've put the workflow in a template file, but kept the QE input part so that user can modify it in the notebook. Let me know whether it is OK now.